stock_manual_transfer/tests/test_stock_manual_transfer.py

```diff
@@ -26,9 +26,9 @@ def create_manual_transfer(self):
         return transfer
 
     def create_manual_transfer_line(self, transfer):
+        self.env.ref("uom.group_uom").write({"users": [(4, self.env.user.id)]})
         with Form(transfer) as tr, tr.transfer_line_ids.new() as line:
             line.product_id = self.product
-            line.product_uom_qty = 1
```

Comment:
Why?

Reply:
It is not needed at this point in the test, as the value is already set to 1. However, we can keep the line even though it is unnecessary.